### PR TITLE
refactor: update meta tags for consistency

### LINKED ISSUE #156: [Bug] Missing `<meta charset="UTF-8">` in Head of All contexts and tests
## Description
The `<meta charset="UTF-8">` tag is missing in the `<head>` section of all context and tests files. This tag is essential for proper character encoding and should be included in all HTML documents.

## Reproduction
1. Open any context HTML file (e.g., [program-page.html]
2. Inspect the` <head> ` section.
3. Notice the absence of the `<meta charset="UTF-8">`  tag.

## Expectation
The `<meta charset="UTF-8">` tag should be present in the <head> section of all context HTML files.
## Screenshots
meta-issue

## Environment

 - Device: (_eg._ iPhone Xs; laptop)
 - OS: (_eg._ iOS 13.5; Fedora 32)
 - Browser: (_eg._ Safari; Firefox)
 - Version: (_eg._ 13; 73)
 - Other info: (_eg._ display resolution, ease-of-access settings)

## Additional context
 This issue affects all context HTML files and should be addressed globally.

## Resolution
- [x] I would be interested in resolving this bug.

### PR DESCRIPTION
**Pull request by Faith-K-commits**
## Description
This pull request fixes #156 missing `<meta charset="UTF-8">` tags in the `<head>` section of all specimen folder HTML files. Adding this tag ensures proper character encoding for the documents.

## Technical details
Added `<meta charset="UTF-8">` tag to the `<head>` section of all context HTML files.

[//]: <> (## Tests)

## Screenshots
Problem:
[
meta-issue
](url)

Solution:
meta_tag_solution

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

For the purposes of this DCO, "license" is equivalent to "license or public domain dedication," and "open source license" is equivalent to "open content license or public domain dedication."
<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by possumbilities** [comment]
Hi @Faith-K-commits and thank you for this work. You've committed changes to files well beyond the scope of this issue, moving files in the structure in the process. Please revert all changes not related to this Issue.

**Comment by Faith-K-commits** [comment]
Hi, thank you for the feedback. Working on it.
Does that include renaming /vocabulary back to /scr?

**Comment by egbadon-victor** [comment]
Hi @Faith-K-commits .

Renaming the the `src` folder to `vocabulary` is only done if you want to implement the vocabulary design system in another project. Since this is the source vocabulary repository, you don't need to do that. Think of this repo as a source of truth for anyone implementing vocabulary.

Hence. the only changes that should be implemented in this PR should be the one highlighted in the screenshot section of your initial comment. I hope this clarifies any questions.

**Comment by Faith-K-commits** [comment]
Thank you so much @egbadon-victor for the clarification.

**Comment by possumbilities** [comment]
Work here would need to address this change in all `html` files within the `specimen` folder and subfolders (including `tests`)

File: specimen/contexts/blog-index.html [hunk 1]
```diff
@@ -118,7 +119,7 @@ <h2>Featured posts</h2>
             <h3 class="title"><a href="#">Open Access in Practice: A Conversation with President Larry Kramer of The Hewlett Foundation</a></h3>
 
             <span class="byline">by <a href="#">Amanda Conway</a>, <a href="#">Ólafur Jónsson</a></span>
-            <span class="categories"><a href=#">Open Culture</a></span>
```
Comment (possumbilities) [inline specimen/contexts/blog-index.html]: this seems unrelated to this Issue?

**Comment by possumbilities** [comment]
This PR seems to include a number of changes that are unrelated.

File: specimen/contexts/blog-index.html (hunk 1 above)
Comment (Faith-K-commits) [reply specimen/contexts/blog-index.html]: Sorry, working on it.

Comment (Faith-K-commits) [reply specimen/contexts/blog-index.html]: I have made the corrections.

File: specimen/contexts/blog-index.html [hunk 2]
```diff
@@ -582,4 +583,4 @@ <h2>Support Our Work</h2>
 <script src="../../src/js/vocabulary.js"></script>
 
 </body>
-</html>
+</html>
```
Comment (possumbilities) [inline specimen/contexts/blog-index.html:586]: End of file missing

File: specimen/index.html [hunk 3]
```diff
@@ -1,20 +1,18 @@
 <!DOCTYPE html>
 <html lang="en-US">
-<head>
-<title>Vocabulary - Specimens</title>
-  
-<meta name="viewport" content="width=device-width, initial-scale=1">
+  <head>
```
Comment (possumbilities) [inline specimen/index.html:3]: Extra spacing changes here

File: specimen/style.css [hunk 4]
```diff
@@ -1,2 +1,2 @@
-@import '../src/css/vocabulary.css' layer(vocabulary);
-@import '../specimen/tests/grid-test.css' layer(vocabulary);
+@import "../src/css/vocabulary.css" layer(vocabulary);
```
Comment (possumbilities) [inline specimen/style.css:1]: unrelated changes here

File: specimen/style.css [hunk 5]
```diff
@@ -1,2 +1,2 @@
...
+@import "../specimen/tests/grid-test.css" layer(vocabulary);
```
Comment (possumbilities) [inline specimen/style.css:2]: unrelated changes here

**Comment by possumbilities** [comment]
Hi @Faith-K-commits 

I also noticed that you mixed upper and lowercase. In some instances you used `utf` and in other you used `UTF`. You should be consistent here. The standard is case-insensitive, but I think for consistency we might want to go with one or the other. There are some arguments that people expect uppercase, though it's not required by the standard, so it might end up with less unforeseen errors later on if we go with uppercase.

**Comment by Faith-K-commits** [comment]
Hello @possumbilities
I have noted that and correcting immediately. 
For the unrelated changes, I am barely editing any other files, I don't know if the issue could be prettier. I am working on it too.

**Comment by possumbilities** [comment]
@Faith-K-commits we aren't currently using prettier on this repository at the moment, so I'd suggest reverting any formatting changes so the PR only reflects the work outlined in the Issue it is addressing.

File: specimen/index.html [hunk 6]
```diff
@@ -1,20 +1,18 @@
...
+    <title>Vocabulary - Specimens</title>
```
Comment (possumbilities) [inline specimen/index.html:4]: extra spacing here.

File: specimen/index.html [hunk 7]
```diff
@@ -1,20 +1,18 @@
...
+    <meta charset="UTF-8" />
```
Comment (possumbilities) [inline specimen/index.html]: extra spacing here

Comment (Faith-K-commits) [reply specimen/index.html]: Okay, let me do the corrections

**Comment by Faith-K-commits** [comment]
I really don't know what's happening, someone can take over.

**Comment by Faith-K-commits** [comment]
I am working on resolving the issues.
